libiocage/lib/errors.py

```diff
@@ -388,7 +388,8 @@ def __init__(self, *args, **kwargs):
 
 class MissingFeature(IocageException, NotImplementedError):
 
-    def __init__(self, feature_name: str, plural: bool=False, *args, **kwargs):
+    def __init__(self, feature_name: str, plural: bool=False,
+                 *args, **kwargs) -> None:
```

Comment:
Please break every argument into a new line. Like such:

```python
def __init__(
    self
    feature_name: str
    plural: bool=False,
    *args
    **kwargs
) -> None:
    pass
```

Reply:
yapf doesn't work for python 3.6, and standard indent can't seem to be able to read minds 😛

Reply:
I'm running this syntax on python3.6. Why should it not work?

Reply:
done.